mobile/src/components/course/components/edit-course-category/edit-course-category.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { FC } from 'react';
+
+import { AppScreenName, DataStatus } from '~/common/enums/enums';
+import {
+  CategoryGetAllItemResponseDto,
+  CourseGetResponseDto,
+} from '~/common/types/types';
+import {
+  BackButton,
+  Button,
+  Dropdown,
+  Spinner,
+  View,
+} from '~/components/common/common';
+import {
+  useAppDispatch,
+  useAppForm,
+  useAppNavigate,
+  useAppSelector,
+  useCallback,
+  useEffect,
+  useFocusEffect,
+} from '~/hooks/hooks';
+import { coursesActions } from '~/store/actions';
+import { courseUpdateCategory as courseUpdateCategoryValidationSchema } from '~/validation-schemas/validation-schemas';
+
+import { DropDownPayload } from './common/common';
+import { getDefaultValueforDropDown } from './helpers/get-categories-options-helper/helpers';
```

Comment:
What is get-categories-options-helper directory?
```suggestion
import { getDefaultValueforDropDown } from './helper/helpers';
```